server/logs/lib.lua

```diff
@@ -0,0 +1,86 @@
+Logs = {}
+Logs.apiUrl= "https://api.fivemanagee.com/logs"
+Logs.apiKey = ""
+
+function Logs:start()
+    Logs.apiKey = GetConvar("FIVEMANAGE_API_KEY", "")
```

Comment:
do we need this start function and calling start at the bottom of the file or could we just fetch the convar directly when creating the variable?